### PR TITLE
Install Python dependencies from requirements.txt

### PR DESCRIPTION
**Pull request by ericvanbuhler**

### CODE REVIEW

File: src/subcommands/app/target/install.ts [hunk 1]
```diff
@@ -37,84 +37,108 @@ export const install = createLeaf({
...
+      async function installPythonDeps(reqFile: string) {
+        const venvRoot = 'venv';
+        try {
+          await spawner.runCommand({
+            exe: 'virtualenv',
+            args: ['--system-site-packages', venvRoot],
+            cwd: '.',
+          });
+        } catch (_) {
+          throw new TerseError('Target does not have virtualenv installed!');
```
Comment (carnesen) [inline src/subcommands/app/target/install.ts]:
```suggestion
          // TODO: More fine-grained error handling
          throw new TerseError('Target does not have virtualenv installed!');
```

File: src/subcommands/app/target/install.ts [hunk 2]
```diff
@@ -37,84 +37,108 @@ export const install = createLeaf({
...
+        }
+        await spawner.runCommand({
+          exe: `${venvRoot}/bin/pip`,
+          args: ['install', '-r', reqFile],
+          cwd: '.',
+        });
+      } // End of install Python dependencies
+      await spinOnPromise(installPythonDeps(reqFile), 'Installing Python dependencies');
+    } else {
+      spinOnPromise(Promise.reject(), `${reqFile} file not found!`);
```
Comment (carnesen) [inline src/subcommands/app/target/install.ts]:
```suggestion
      await spinOnPromise(Promise.reject(), `${reqFile} file not found!`);
```

**Comment by carnesen** [comment]
This "suggested change" feature is news to me. Seems like you can just click "commit suggestion" if you like my changes.

**Comment by ericvanbuhler** [comment]
> This "suggested change" feature is news to me. Seems like you can just click "commit suggestion" if you like my changes.

Let's give it a try!

**Comment by carnesen** [comment]
I need this for something i'm working on. Gonna merge!

**Pull request merged**